### PR TITLE
test(kubeflow): add testgrid config

### PR DESCRIPTION
**Pull request by Bobgy**
Part of https://github.com/kubernetes/test-infra/issues/14343

/assign @chaodaiG

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -15,3 +17,7 @@ dashboard_groups:
     - googleoss-esp-v2-presubmit
     - googleoss-esp-v2-periodic
     - googleoss-en-server
+  - name: kubeflow
```
[the remark this answers is not in the document]
Comment (chases2) [reply testgrid/config.yaml]: Now that it's renamed, I think it's fine. The kubernetes/test-infra repository has some standards regarding group prefixes, but Transfigure tests should cover it.

**Review by chases2 — APPROVED** [review]
Looks good to me! Feel free to drop the hold when ready.
/hold

nit: please squash the commits together into one

**Comment by Bobgy** [comment]
Thank you!
/unhold

**Comment by Bobgy** [comment]
Is it possible to change merge method to default to squash?

Let me do a manual squash this time.

**Comment by Bobgy** [comment]
Hmmm, tide was faster than me

**Comment by Bobgy** [comment]
To prevent this from happening, we add a config like this for kubeflow: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/64354663fb618b4b53e7f0aa592bff24f96be06b/prow/oss/config.yaml#L26

Shall we consider adding the same for this repo?

**Pull request merged**